Source/WebKit/Shared/ApplePay/mac/WebPaymentCoordinatorProxyMac.mm

```diff
@@ -109,7 +110,9 @@
 
             ASSERT(!paymentCoordinatorProxy->m_sheetWindow);
             paymentCoordinatorProxy->m_sheetWindow = [NSWindow windowWithContentViewController:viewController];
-
+#if HAVE(UI_GLASS_EFFECT)
```

Comment:
```suggestion
#if HAVE(LIQUID_GLASS)
```

`HAVE(UI_GLASS_EFFECT)` is only true on iOS and tvOS (it refers to UIKit's `UIGlassEffect`).